examples/experimental/README.md

```diff
@@ -1 +0,0 @@
-All notebooks put in this folder will be deleted if left unattended for 14 days or more. They exist purely for experimental purposes (and the ability to temporarily communicate experimental results to the group). As such, they are NOT guaranteed to work or checked for code style/functionality in any way. For more robust/tested/permanent notebooks, see other folders outside this one.
```

Comment:
So not just clean up of old experiments, but also removing overall concept of experiments. Is it correct?

Reply:
The directory will come back when someone puts in an experiment

Reply:
shall we keep readme in this case?

Reply:
I would agree to do so